src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj

```diff
@@ -62,6 +62,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Azure.Data.Tables" Version="12.8.3" />
```

Comment:
What work is remaining, if any, to remove `Microsoft.Azure.Cosmos.Table` package reference?

Reply:
We also need to migrate the Scale Metrics https://github.com/Azure/azure-functions-host/issues/10166. I thought on splitting in 2 different PRs.